packages/app/cypress/e2e/specs.cy.ts

```diff
@@ -455,177 +455,6 @@ describe('App: Index', () => {
     viewportHeight: 768,
     viewportWidth: 1024,
   }, () => {
-    context('project with storybook', () => {
```

Comment:
Arn't we adding story spec generation in a future release? IF so, should we skip these tests with a link tot he followup work and just update the UI to remove this spec option?

Reply:
If not a soon followup, remove this code LGTM.